submissions/kuslosolodka/html-css-popup/index.html

```diff
@@ -0,0 +1,143 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Popup</title>
+    <link rel="stylesheet" href="styles/style.css" />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Raleway&display=swap"
+      rel="stylesheet"
+    />
+  </head>
+  <body>
+    <input type="checkbox" id="popup-btn" class="popup-checkbox" />
+    <label class="popup-label" for="popup-btn">
+      <img class="popup-icon" src="assets/icons/popup-button.png" />
+    </label>
+    <nav class="popup-menu">
+      <ul class="icons">
+        <li>
+          <a href="#" class="menu-icon"
+            ><img
+              class="icon-photo"
+              src="assets/icons/calendar.png"
+            />Calendar</a
+          >
+        </li>
+        <li>
+          <a href="#" class="menu-icon"
+            ><img class="icon-photo" src="assets/icons/mail.png" />Mail</a
```

Comment:
The `alt` attribute is required  so if you want to set an empty value use this example:
```HTML
<img alt="">
```